lighthouse-core/test/fraggle-rock/gather/navigation-runner-test.js

```diff
@@ -173,15 +173,21 @@ describe('NavigationRunner', () => {
       await expect(run()).rejects.toBeTruthy();
     });
 
-    it('should navigate as many times as there are navigations', async () => {
```

Comment:
It might make sense to preserve this test since the logic is still there, we just can't generate a config for it through conventional means.